### PR TITLE
Add additional area labels that have been added recently

### PR DESCRIPTION
**Pull request by jeffschwMSFT**
There are a few area labels that we are using that do not have owners assigned.  I did my best to identify the right people.

cc @danmoseley @tommcdon @marek-safar

### CODE REVIEW

**Review by tommcdon — APPROVED** [review]
LGTM

**Comment by danmoseley** [comment]
@maryamariyan does the bot already know about these in its training?

**Comment by danmoseley** [comment]
I may be able to answer that: yes it should, as these labels are not new with the possible exception of area-System.Diagnostics.TraceSource whose earliest use may have been March.

**Comment by jeffschwMSFT** [comment]
Most of these are not 'new', but they are not in our owners.  I think I missed one: area-System.Speech.  I will push another round tomorrow (after I double check a few more areas).

**Comment by danmoseley** [comment]
Ah yes- you can put me as the owner of that one. The effort seems to be negligible so far. Also, we already trained with that one too.

File: docs/area-owners.md [hunk 1]
```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
...
+| area-System.Diagnostics.EventLog               | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
+| area-System.Diagnostics.PerformanceCounter     | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
```
Comment (ericstj) [inline docs/area-owners.md:82]:
```suggestion
| area-System.Diagnostics.EventLog               | @ericstj | @Anipik @ViktorHofer                    |                                                                                                                                                                                                                                                                                           |
| area-System.Diagnostics.PerformanceCounter     | @ericstj | @Anipik @ViktorHofer                    |                                                                                                                                                                                                                                                                                           |
```

File: docs/area-owners.md [hunk 2]
```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
...
 | area-System.Diagnostics                        | @tommcdon     | @tommcdon @Anipik @ViktorHofer                      | Sub-Areas: <ul><li>System.Diagnostics.EventLog - @Anipik @ViktorHofer</li></ul>                                                                                                                                                                                                           |
+| area-System.Diagnostics-coreclr                | @tommcdon     | @tommcdon @Anipik @ViktorHofer                      | Sub-Areas: <ul><li>System.Diagnostics.EventLog - @Anipik @ViktorHofer</li></ul>                                                                                                                                                                                                           |
```
Comment (ericstj) [inline docs/area-owners.md:78]: It's unclear what these now represent, I'm not seeing anything left owned by my team.
``` suggestion
| area-System.Diagnostics                        | @tommcdon     | @tommcdon                       |                                                                                                                                                                                                            |
| area-System.Diagnostics-coreclr                | @tommcdon     | @tommcdon                       |                                                                                                                                                                                                            |
```

File: docs/area-owners.md [hunk 3]
```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
...
 | area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - @Anipik @ViktorHofer</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li><li>System.Diagnostics.TraceSource</li></ul>                                                     |
+| area-System.Diagnostics.TraceSource            | @jeffhandley  | @buyaa-n @joperezr @krwq                            | Consultants: @tquerec @josephisenhour                                                                                                                                                                                                                                                     |
```
Comment (ericstj) [inline docs/area-owners.md:85]:
```suggestion
| area-System.Diagnostics.TraceSource            | @ericstj  | @Anipik @ViktorHofer                            |                                                                                                                                                                                                                                                      |
```

**Review by ericstj — COMMENTED** [review]
Can you please make these adjustments?  Also when creating new labels we typically want to go back and reassign past issues/PRs (even closed ones) so that we can retrain the bot.  Was this done?  cc @maryamariyan

File: docs/area-owners.md [hunk 4]
```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
...
 | area-System.Diagnostics.Process                | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - @Anipik @ViktorHofer</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li><li>System.Diagnostics.TraceSource</li></ul>                                                     |
```
Comment (ericstj) [inline docs/area-owners.md:84]: Looks like this is the only one that's still spanning teams.  Would it make sense to give DiagnosticSource its own label?
```suggestion
| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li> 
```

**Comment by jeffschwMSFT** [comment]
> Can you please make these adjustments? 

Will do.  I have at least one more tag to update tomorrow.

> Also when creating new labels we typically want to go back and reassign past issues/PRs (even closed ones) so that we can retrain the bot. Was this done? 

These are not new, just were never documented.  Though I agree with your statement.

**Comment by maryamariyan** [comment]
@jeffschwMSFT once older issues get updated with the proper labels please let me know so I can retrain with new changes.

**Pull request merged**